kount/request.py

```diff
@@ -139,6 +143,7 @@ def khash_payment_encoding(self, enabled=True):
             self.params["PENC"] = "KHASH"
         else:
             self.params["PENC"] = "MASK"
```

Comment:
The `else` part here should set `PENC = NONE`. The `MASK` option is meant to be for credit cards only and the function, implementing the masking option should be as an alternative to the standard `setPayment(Payment p)` function.